kartograf/rpki/fetch.py

```diff
@@ -102,6 +103,9 @@ def validate_rpki_db(context):
 
     tal_options = [item for path in data_tals(context) for item in ('-t', path)]
 
+    cpu_count = os.cpu_count()
+    threads = cpu_count if cpu_count else 4
```

Comment:
Since [#98](https://github.com/asmap/kartograf/pull/98) is merged now, please put this threads count stuff into some util function that is shared between the two different places.